SubmodelRepositoryServiceSpecification/V3.1_SSP-005.yaml

```diff
@@ -0,0 +1,214 @@
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/accepted'
+        '400':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/bad-request'
+        '401':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/unauthorized'
+        '403':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/forbidden'
+        '500':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/internal-server-error'
+  /status/{handleId}:
+    get:
+      tags:
+        - Async Bulk Status API
+      summary: Returns the status of an asynchronously invoked bulk operation
+      operationId: BulkGetAsyncStatus
+      x-semanticIds:
+        - https://admin-shell.io/aas/API/BulkGetAsyncStatus/3/1
+      parameters:
+        - in: path
+          name: handleId
+          schema:
+            $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/parameters/HandleId'
+          required: true
+      responses:
+        '200':
+          description: Bulk operation result object containing information that the 'executionState' is still 'Running'
+          headers:
+            Retry-After:
+              schema:
+                type: integer
+              required: false
+              description: The number of seconds to wait before retrying the request.
+          content:
+            application/json:
+              schema:
+                $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/schemas/BaseOperationResult'
+        '302':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/found'
+        '401':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/unauthorized'
+        '403':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/forbidden'
+        '404':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/not-found'
+        '500':
+          $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/responses/internal-server-error'
+  /result/{handleId}:
+    get:
+      tags:
+        - Async Bulk Result API
+      summary: Returns the result object of an asynchronously invoked bulk operation
+      operationId: BulkGetAsyncResult
+      parameters:
+        - in: path
+          name: handleId
+          schema:
+            $ref: 'https://api.swaggerhub.com/domains/Plattform_i40/Part2-API-Schemas/V3.0.1#/components/parameters/HandleId'
+          required: true
+      responses:
```

Comment:
Regarding the Bulk Result, wouldn't it be more meaningful to use [response code 207](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/207) for bulk request results and provide individual response for elements? See also [Zalando REST API best practices](https://opensource.zalando.com/restful-api-guidelines/#152) 

For example if I have 100 submodels to create and only 1 of them is duplicate how the client can know the issue and handle it?